app/views/users/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+  <body>
+    <h1><%= @message %></h1>
+  </body>
+</html>
```

Comment:
- [ ] Same goes here, please leave only `h1 `tag and delete the rest in all views except `app/views/layouts/application.html.erb` 😉

Reply:
Same as previous